astropy/io/fits/util.py

```diff
@@ -824,12 +824,24 @@ def _array_to_file_like(arr, fileobj):
     `numpy.ndarray.tofile`).
     """
 
+    if len(arr) == 0:
```

Comment:
@MSeifert04 I added a comment - this is just a shortcut to avoid executing the rest of the function.